nomad/structs/structs.go

```diff
@@ -916,6 +938,10 @@ func (u *UpdateStrategy) Rolling() bool {
 const (
 	// PeriodicSpecCron is used for a cron spec.
 	PeriodicSpecCron = "cron"
+
+	// PeriodicSpecTest is only used by unit tests. It is a sorted, comma
+	// seperated list of unix timestamps at which to launch.
+	PeriodicSpecTest = "test"
```

Comment:
Given this behavior, maybe it should just be called "fixed" since it runs at fixed times. Otherwise, maybe pick something less likely to be accidentally used, e.g. "_internal_test"